packages/flutter_tools/lib/src/plugins.dart

```diff
@@ -213,6 +213,14 @@ class Plugin {
       return <String>[errorMessage];
     }
 
+    if (!usesOldPluginFormat && !usesNewPluginFormat) {
+      const String errorMessage =
+          'The flutter.plugin section format at the `pubspec.yaml` is not correct. '
```

Comment:
What about: 
```
Cannot find the `flutter.plugin.platforms` key in the `pubspec.yaml` file.
```

We want to push users to use the new format.

Reply:
OK.